pkg/sentry/csr/csr.go

```diff
@@ -212,3 +213,81 @@ func newSerialNumber() (*big.Int, error) {
 	}
 	return serialNum, nil
 }
+
+type CredentialRequests struct {
+	PrivateKey  *certs.PrivateKey
+	Certificate *x509.CertificateRequest
+}
+
+func PEMCredentialRequestsFromFiles(certPem, keyPem []byte) (*CredentialRequests, error) {
```

Comment:
Where is this being used apart from test?

Reply:
Remove it.